src/libraries/Common/src/Interop/Windows/NCrypt/Interop.NCryptDeriveSecretAgreement.cs

```diff
@@ -43,7 +43,7 @@ internal enum SecretAgreementFlags
 
             if (error != ErrorCode.ERROR_SUCCESS)
             {
-                throw error.ToCryptographicException();
+                throw ((int)error).ToCryptographicException();
```

Comment:
Is this called from a context that doesn't have the CngHelpers class that the extension method moved to?  (Or was it point-in-time thing and can be reverted?)

Reply:
The latter.